ui/pages/sample/components/sample-network-display.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
If there is a new component for redesign we add it to Multichain directory and if it's a base component, use Component-library directory cc @georgewrmarshall

Reply:
Great call out @NidhiKJha! I think non-DS components can either live in app/ or ui/ but I know this might be contrary to ways I've seen it done in the past